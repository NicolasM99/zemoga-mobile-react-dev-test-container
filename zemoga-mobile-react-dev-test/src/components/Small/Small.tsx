import React, { FC } from 'react';

import { Text } from 'react-native';

import { ISmall } from './@types/small';

const Small: FC<ISmall> = ({ children, testID }: ISmall) => {
  return (
    <Text testID={testID} style={{ fontSize: 12 }}>
      {children}
    </Text>
  );
};

export default Small;
