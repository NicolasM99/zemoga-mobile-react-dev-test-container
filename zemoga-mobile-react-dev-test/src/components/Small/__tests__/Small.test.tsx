import { render, screen } from '@testing-library/react-native';

import Small from '../Small';

describe('<Small /> unit tests', () => {
  it('Should mount correctly the component', () => {
    render(<Small>Lorem ipsum small</Small>);
    expect(screen).toMatchSnapshot();
  });
});
