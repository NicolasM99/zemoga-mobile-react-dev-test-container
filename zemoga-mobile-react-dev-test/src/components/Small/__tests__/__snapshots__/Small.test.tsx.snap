// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Small /> unit tests Should mount correctly the component 1`] = `
<Text
  style={
    Object {
      "fontSize": 12,
    }
  }
>
  Lorem ipsum small
</Text>
`;
